src/pages/players/[id]-[username].astro

```diff
@@ -33,55 +33,68 @@ const highestLeague = player.leaderboard_entries.reduce(
   (acc, entry) => (entry.points > acc.points ? entry : acc),
   player.leaderboard_entries[0]
 )
+
+const playerActivity = (await PlayersApi.getPlayerStatisticsActivity({ playerId })) as any as PlayerActivityStats
 ---
 
 <Layout title={player.nickname}>
   <div class="w-full border-b border-gray-700/50 bg-gray-800/50 backdrop-blur-lg">
-    <div class="mx-auto flex max-w-screen-md flex-wrap items-center gap-4 px-4 py-4">
+    <div class="mx-auto flex max-w-screen-lg flex-wrap items-center gap-4 px-4 py-4">
       <RankedBadge entry={highestLeague} class="w-20" client:load />
       <h1 class="flex-auto text-2xl font-bold text-gray-50">{player.nickname}</h1>
+    </div>
+  </div>
+  <section class="relative mx-auto flex max-w-screen-lg flex-col gap-6 px-4 py-8 lg:flex-row">
```

Comment:
In my previous PR to unify the spacing, I used `md:px-7` to match the one of the header. Maybe worth adding?